### PR TITLE
 grammatical and spelling errors were resolved

### LINKED ISSUE #5: Grammatical and spelling errors in the Terms of Service and Privacy Policy section
Grammatical and spelling errors in the Terms of Service and Privacy Policy section, I want to resolve them, Kindly assign me this issue

Add Screenshot

spelling

### PR DESCRIPTION
**Pull request by HimanshuNarware**
#Description 
Grammatical and spelling errors were resolved from the website successfully. It will improve the readability of the website as well as the user experience.

# issue code 
#5

#Screenshot

### CODE REVIEW

**Comment by kailashchoudhary11** [comment]
@HimanshuNarware Don't make a PR until the issue is assigned to you.

File: components/homepage/About/About.jsx [hunk 1]
```diff
@@ -14,17 +14,17 @@ const About=()=>{
                             <div className={[style.feature__heading, style.left].join(' ')}>
                                 <h2>Royalty Free</h2>
                                 <p>Enjoy Royalty Free awesome music for free
-                                We provide all of our services free
+                                We provide all of our services for free
                                 Login and use vovoca for free forever</p>
                             </div>
```
Comment (varun-singhh) [inline components/homepage/About/About.jsx:19]: Replace `Enjoy Royalty Free awesome music for free
                                We provide all of our services for free
                                Login and use vovoca for free forever`

with 

`Experience the rhythm, without spending a dime,
Unlock endless melodies, vovoca's free for a lifetime!`

File: components/homepage/About/About.jsx [hunk 2]
```diff
@@ -14,17 +14,17 @@ const About=()=>{
...
                         <div className={style.feature}>
                         <div className={[style.feature__heading, style.left].join(' ')}>
                                 <h2>No Credits Required</h2>
-                                <p>You dont even have to pay us or give us credit
+                                <p>You don't even have to pay us or give us credit
                                 for using our music anywhere, in your songs,
-                                vidoes or office presentation</p>
+                                videos or an office presentation</p>
                             </div>
                         </div>
```
Comment (varun-singhh) [inline components/homepage/About/About.jsx:29]: Replace `You don't even have to pay us or give us credit
                                for using our music anywhere, in your songs,
                                videos or an office presentation` 

with 

`Let the melodies soar, no strings attached,
Vovoca's music, freely used, without credit or cash.`

File: components/homepage/About/About.jsx [hunk 3]
```diff
@@ -35,16 +35,16 @@ const About=()=>{
                     <div className={style.feature}>
                     <div className={[style.feature__heading, style.right].join(' ')}>
                                 <h2>Filter Music</h2>
-                                <p>Login to utilise complete set of features
-                                of vovoca. You can then easily filter
+                                <p>Login to utilize a complete set of features
+                                of Vovoca. You can then easily filter
                                 our music category wise</p>
                             </div>
```
Comment (varun-singhh) [inline components/homepage/About/About.jsx:41]: Replace ` Login to utilize a complete set of features
                                of Vovoca. You can then easily filter
                                our music category wise` 

with 

`Unlock Vovoca's full potential with a single login,
Effortlessly browse and filter, music at your command`

File: components/homepage/About/About.jsx [hunk 4]
```diff
@@ -35,16 +35,16 @@ const About=()=>{
...
                         </div>
                         <hr className="divider"></hr>
                         <div className={style.feature}>
                         <div className={[style.feature__heading, style.right].join(' ')}>
                                 <h2>Upload Music</h2>
-                                <p>You can also share your work and talent to whole
+                                <p>You can also share your work and talent with the whole
                                 world for free. Sign in and upload your
                                 created music with the world</p>
                             </div>
```
Comment (varun-singhh) [inline components/homepage/About/About.jsx:50]: Replace `You can also share your work and talent with the whole
                                world for free. Sign in and upload your
                                created music with the world`

 with 

`Unleash your artistry, let your talent shine bright,
Share your creations with the world, day or night.
Vovoca welcomes your music, for free, with delight`

File: components/homepage/Footer/Footer.jsx [hunk 5]
```diff
@@ -9,7 +9,7 @@ const Footer = () => {
           <div className={style.get__in__touch}>
             <h2>Get in Touch</h2>
             <p>
-              Want to work, collaborate with us or you have any idea related to
+              Want to work, or collaborate with us or do you have any ideas related to
               our project? Feel Free to contact us anytime. We are just a click
               away
             </p>
```
Comment (varun-singhh) [inline components/homepage/Footer/Footer.jsx:15]: Replace with `Join forces with us, let's create something grand,
Collaboration awaits, reach out with a click at hand.
Share your ideas, let's shape our project, hand in hand.`

File: components/homepage/Header/Header.jsx [hunk 6]
```diff
@@ -126,8 +126,8 @@ const Header = () => {
             #Tune your work with Royalty Free Music
           </h1>
           <p className={style.banner__sub__heading}>
-            Welcome to vovoca, one stop solution for all you musical needs, we
-            provide royalty free music so that your creativity never stops.
+            Welcome to Vovoca, one-stop solution for all your musical needs, we
+            provide royalty-free music so that your creativity never stops.
           </p>
           {authenticated === true ? (
```
Comment (varun-singhh) [inline components/homepage/Header/Header.jsx:132]: Replace content in p tag with `Enter Vovoca's realm, where music knows no bounds,
Fuel your creativity with royalty-free sounds.
Unleash your imagination, let your creations astound`

File: pages/developers.js [hunk 7]
```diff
@@ -34,10 +34,10 @@ console.log(data);
                 <article className={styles.testimonial__article}>
                   <div className={styles.testimonial__content}>
                     <p>
-                      Though this is a simple music streaming and downloading
+                    Though this is a simple music streaming and downloading
                       website, but BTS it was a roller coaster ride while
```
Comment (varun-singhh) [inline pages/developers.js:38]: Replace content in p tag with `From humble beginnings to an extraordinary ride,
BTS, we conquered bugs and reached new heights.
Teamwork and dedication, our driving force with pride,
Hard work paid off, now Vovoca stands by your side.`

File: pages/privacy-policy.js [hunk 8]
```diff
@@ -35,7 +35,7 @@ const privacyPolicy = () => {
                         </div>
                     </div>
                     <p id="1b" id="1" className={style.modal__button__para}>
-                        Vovoca is a project under team DevBucket. We have putted our heart and soul in it to bring it to you for free. We our free today because of several open-source technologies out there that we have used all togethor to bring you Vovoca. We therefore aim to bring you vovoca as a open-source program too after completion of all the basic structure and interface. Our backend APIs are currently hosted on heroku and frontend on vercel while in development mode.
+                    Vovoca is a project under team DevBucket. We have put our heart and soul into it to bring it to you for free. We are free today because of several open-source technologies out there that we have used all together to bring you Vovoca. We therefore aim to bring you Vovoca as a open-source program too after completion of all the basic structure and interface. Our backend APIs are currently hosted on Heroku and frontend on vercel while in development mode.
                     </p>
```
Comment (varun-singhh) [inline pages/privacy-policy.js:39]: Replace text with `Welcome to Vovoca, a remarkable endeavor by the passionate team at DevBucket. Our unwavering dedication has fueled the creation of this platform, offered to you completely free of charge. We owe our freedom today to the incredible open-source technologies that have seamlessly converged within Vovoca. With utmost commitment, our ultimate goal is to transform Vovoca into an open-source program, granting access to its inner workings once the foundation and interface are perfected. During our development journey, Vercel graciously hosts both our backend APIs and frontend, ensuring a seamless and efficient experience. Together, let's embrace the power of collaboration and embark on a harmonious musical voyage with Vovoca.`

File: pages/privacy-policy.js [hunk 9]
```diff
@@ -46,7 +46,7 @@ const privacyPolicy = () => {
...
                     <p id="2" className={[style.modal__button__para, style.none].join(' ')}>
-                        We are more than aware of your private data. We have taken care of end to end encryption as much as we could have. All your passwords are hashed before going into our database. We however suggest you to not to use your common password on this website. You can use auto password gernerator on any browser for the same. Team DevBucket does not hold any guarantee on any possible leakage of private data or your music. We However assure you that we have followed all the best practices to secure your data
+                    We are more than aware of your private data. We have taken care of end-to-end encryption as much as we could have. All your passwords are hashed before going into our database. We however suggest you to not to use your common password on this website. You can use an auto password generator on any browser for the same. Team DevBucket does not hold any guarantee on any possible leakage of private data or your music. We However assure you that we have followed all the best practices to secure your data
                     </p>
```
Comment (varun-singhh) [inline pages/privacy-policy.js:50]: replace text with `At Vovoca, your privacy is our utmost priority. We understand the sensitivity of your personal information and have taken extensive measures to safeguard it. Through meticulous implementation of end-to-end encryption, we strive to ensure that your data remains protected throughout its journey. To provide an added layer of security, all passwords are securely hashed before being stored in our database. As an extra precaution, we strongly recommend refraining from using commonly used passwords on this platform. Instead, leverage the convenience of auto password generators available on various browsers. While Team DevBucket cannot provide an absolute guarantee against any potential data breaches or music leaks, we assure you that we have diligently adhered to industry best practices to fortify the security of your data. Rest assured, your privacy is in safe hands at Vovoca.`

File: pages/privacy-policy.js [hunk 10]
```diff
@@ -57,7 +57,7 @@ const privacyPolicy = () => {
                         </div>
                     </div>
                     <p id="3" className={[style.modal__button__para, style.none].join(' ')}>
-                        We have used most secured and open sourced programs to host our web app currently. We are using heroku and node and postgreSQL for our database. Therefore any data breach over heroku may lead to  data leakage on our website. In any such case, Vovoca or DevBucket does not holds any guarantee. We are currently working day and night and implementing methods to make vovoca a more secure app for your music and email and password. 
+                    We have used the most secured and open-sourced programs to host our web app currently. We are using Heroku  node and PostgreSQL for our database. Therefore any data breach over Heroku may lead to data leakage on our website. In any such case, Vovoca or DevBucket does not hold any guarantee. We are currently working day and night and implementing methods to make Vovoca a more secure app for your music and email and password.  
                     </p>
```
Comment (varun-singhh) [inline pages/privacy-policy.js:61]: replace content with 

At Vovoca, we take the security of your data seriously and have implemented stringent measures to protect your information. Our web app is hosted using the highly secure and open-source programs, Vercel, Node, and PostgreSQL, ensuring a robust infrastructure for data storage. However, it is important to note that in the unlikely event of a data breach on the Vercel platform, there is a possibility of data leakage on our website.

While we strive to maintain the highest level of security, we cannot provide an absolute guarantee against data breaches. In the event of such an occurrence, neither Vovoca nor DevBucket can be held responsible. However, please be assured that we are working tirelessly, dedicating ourselves day and night to implementing additional methods and security measures to further enhance the protection of your music, email, and password within the Vovoca app.

We value your trust in us, and the privacy and security of your data remain our top priority. We will continue to make every effort to ensure the utmost security of your information and provide you with a safe and secure platform for your musical experience.

Thank you for choosing Vovoca, and we appreciate your understanding and support as we continually strive to improve our security practices.

**Review by varun-singhh — CHANGES_REQUESTED** [review]

**Comment by HimanshuNarware** [comment]
> @HimanshuNarware Don't make a PR until the issue is assigned to you.

sorry sir

**Pull request closed without merging**